### ISSUE TITLE
"Cited on" backref can overlap with reference number

### ISSUE THREAD

**Issue by jgoeders** [BYU-Engineering/thesis_template#7]
The "cited on" text used in the references (`backref`) is configured to print in the margin via a [refnote](https://github.com/BYU-Engineering/thesis_template/blob/d572152308b505db682a6a624aff1bd5f7107f4e/src/byubib.bbx#L170) command.  However, the reference numbers are also printed in the margin.  This can cause an overlap when a reference is cited several times:

<img width="600" alt="image" src="https://github.com/BYU-Engineering/thesis_template/assets/1851142/a967252f-a87b-4d87-932e-a2620f932706">

**Comment by jgoeders** [comment]
I'm not sure how to fix this yet.  I suppose the two options are:
1. Shrink the width of the "cited on" text in the margins.
2. Change the bibliography printing so that the item number isn't in the margin.  (The references text would no longer line up with the document text)

## Approach 1 
I can fix this from the student's .tex file like so:
```tex
\addtolength{\marginparsep}{0.75cm}
\addtolength{\marginparwidth}{-0.75cm}
 
\printbibliography[title=References]
 
\clearpage
\addtolength{\marginparsep}{-0.75cm}
\addtolength{\marginparwidth}{0.75cm}

```

But I can't figure out how to do this in the template class. Adding these commands to `\defbibenvironment` doesn't seem to work...

## Approach 2

This can be accomplished by changing the left margin in `\defbibenvironment` [here](https://github.com/BYU-Engineering/thesis_template/blob/d572152308b505db682a6a624aff1bd5f7107f4e/src/byubib.bbx#L18).

```
 \setlength{\leftmargin}{0pt}%
```

Increasing the `0pt` to something larger will shift the references over, but they will no longer be in line with the document text.

@BYU-Engineering What are your thoughts?

**Comment by jgoeders** [comment]
Fixed in 0802f5807e6c86f9c5e923c697dbb1b5e0471090
